peer/brontide.go

```diff
@@ -2431,25 +2431,29 @@ out:
 				continue
 			}
 
-			// We'll query the localChanCfg of the new channel to determine the
-			// minimum HTLC value that can be forwarded. For the maximum HTLC
-			// value that can be forwarded and fees we'll use the default
-			// values, as they currently are always set to the default values
-			// at initial channel creation. Note that the maximum HTLC value
-			// defaults to the cap on the total value of outstanding HTLCs.
-			//
-			// TODO(guggero): We should instead pass in the current
-			// forwarding policy from the funding manager to avoid
-			// needing us to update the link once the channel is
-			// announced to the network with custom user values.
-			fwdMinHtlc := lnChan.FwdMinHtlc()
-			defaultPolicy := p.cfg.RoutingPolicy
+			// We'll query the channel DB for the new channel's
+			// initial forwarding policies to determine the policy
+			// we start out with.
+			db := p.cfg.ChannelDB
+			initialPolicy, err := db.GetInitialForwardingPolicy(
```

Comment:
So for private channel or zero conf channel, since it's deleted this will always give an error?

Reply:
The data is deleted only after we've announced the channel to the network, which happens after adding it to `brontide`.